persistence/pre_persistence/block.go

```diff
@@ -21,7 +22,7 @@ func (b *Block) ValidateBasic() types.Error {
 }
 
 func (bh *BlockHeader) ValidateBasic() types.Error {
-	if bh.NetworkID == "" {
+	if bh.NetworkId == "" {
```

Comment:
We should look into shared utils sometime further down the integration path. As once previously pointed out by Shans in one of my P2P PR, let's use:
```suggestion
  if utils.IsEmpty(bhNetworkId) {
```

* Less error prone
* Reusable
* More semantic

Reply:
Add it as a protocol hour backlog discussion to not lose it